helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -56,10 +56,6 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "logstash"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
-    FEATURE_EVENT_PRISON_VISIT_BOOKED: true
```

Comment:
these must be redundant given the switch code defaults to true anyway. Only do this for the domains I know how to test for now